cpp/src/io/parquet/page_enc.cu

```diff
@@ -103,7 +108,8 @@ __global__ void __launch_bounds__(512) gpuInitPageFragments(PageFragment *frag,
 
   frag_init_state_s *const s = &state_g;
   uint32_t t                 = threadIdx.x;
-  uint32_t start_row, nrows, dtype_len, dtype_len_in, dtype;
+  uint32_t start_row, dtype_len, dtype_len_in, dtype;
+  size_type start_value_idx, nvals;
```

Comment:
![image](https://user-images.githubusercontent.com/16005690/91606310-b3099480-e926-11ea-8839-fdb5d4ab1133.png)

I think all of these can be declared at the point of use.